changelog.md

```diff
@@ -1,10 +1,10 @@
 # Changelog
 
-## v0.6.2 | t.b.d.
+## v0.6.2 | 2020-09-21
 
 * Support plotting Z-axis scans. Z-axis scans would previously throw an exception due to how the physical dimensions were fetched. This issue is now resolved.
 * Add slicing (by time) for `FDCurve`.
-* Add widget to slice Fd's with from Jupyter Notebooks.
+* Add widget to graphically slice `FDCurve` in Jupyter Notebooks. It can be opened by `pylake.FdRangeSelector(fdcurves)`. For more information see docs tutorials section: Notebook widgets.
```

Comment:
Consider adding a link to the new docs page.
```suggestion
* Add widget to graphically slice `FDCurve` in Jupyter Notebooks. It can be opened by calling `pylake.FdRangeSelector(fdcurves)`. For more information see docs tutorials section: Notebook widgets.
```

Reply:
You mean markdown style with full URL: [Notebook widgets](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/nbwidgets.html)
Or rst style (but then it'll only work on rtd)?

Reply:
Markdown style. That will work when Sphinx imports it.